macOS/DuckDuckGo/DuckDuckGoAppStore.entitlements

```diff
@@ -56,7 +56,7 @@
 	<key>keychain-access-groups</key>
 	<array>
 		<string>$(DBP_APP_GROUP)</string>
-		<string>$(AppIdentifierPrefix)com.duckduckgo.mobile.ios</string>
```

Comment:
NEW: This is supposed to fix keychain dialogs _for good_ (famous last words). I found that entitlements files hardcode the main production bundle ID in keychain access groups definition, so if we parametrize it, each build should finally be using its own keychain group.